workitem/field_definition.go

```diff
@@ -63,6 +63,7 @@ type FieldType interface {
 // FieldDefinition describes type & other restrictions of a field
 type FieldDefinition struct {
 	Required    bool      `json:"required"`
+	IsReadOnly  bool      `json:"is_read_only"`
```

Comment:
Why not just `ReadOnly`? We have `Required` and not `IsRequired`.

Reply:
In terms of consistency you're right @baijum . I always think it is more expressive for boolean values to have an `Is` infront. It is a good convention. I'll see what I will do about this. Maybe change `Required` to `IsRequired`? Don't know yet.

Reply:
I go with `ReadOnly`.